midend/local_copyprop.cpp

```diff
@@ -228,6 +260,26 @@ IR::AssignmentStatement *DoLocalCopyPropagation::postorder(IR::AssignmentStateme
     return as;
 }
 
+IR::IfStatement *DoLocalCopyPropagation::postorder(IR::IfStatement *s) {
```

Comment:
Yes, for `p4_16_samples/hit_ebpf.p4`, the compiler throws a bug of the if statement will a null `IfTrue` expression. The `IfStatement` is validated before the `ElimDead` is called on this object, which results in the compiler exiting with a bug.